default/python/AI/expansion_plans/expansion_plans_interface.py

```diff
@@ -0,0 +1,32 @@
+from typing import FrozenSet, Iterable, OrderedDict, Tuple
+
+from common.fo_typing import PlanetId, SpeciesName
+from freeorion_tools.caching import cache_for_current_turn
+
+_the_planner = None
+
+
+@cache_for_current_turn
+def colonies_targeted() -> FrozenSet[PlanetId]:
+    return _the_planner.colonies_targeted()
+
+
+@cache_for_current_turn
+def outposts_targeted() -> FrozenSet[PlanetId]:
+    return _the_planner.outposts_targeted()
+
+
+def get_colonisable_planet_ids(include_targeted: bool = False) -> OrderedDict[PlanetId, Tuple[float, SpeciesName]]:
```

Comment:
I'd say I'd prefer to have 2 methods here, or always use keyword argument when we call them.

When I see `get_colonisable_planet_ids(True)`, I have no idea what it means and need to check the implementation.

```python
def get_colonisable_planet_ids() -> OrderedDict[PlanetId, Tuple[float, SpeciesName]]: ...
def get_colonisable_planet_ids_including_targeted() -> OrderedDict[PlanetId, Tuple[float, SpeciesName]]: ...
```

Alt
```python
= get_colonisable_planet_ids(include_targeted=True)
```